api/src/services/restaurants/restaurants.js

```diff
@@ -26,7 +36,18 @@ export const updateRestaurant = ({ id, input }) => {
   requireAuth()
   return db.restaurant.update({
     data: {
-      ...input,
+      name: input.name,
+      brandColor: input.brandColor,
+      address: {
+        upsert: {
+          create: {
+            ...input.address,
+          },
+          update: {
```

Comment:
We can just do an `update` instead of `upsert` (the address is created when the restaurant is).